### ISSUE TITLE
Docker Socket invalid URL

### ISSUE THREAD

**Issue by VincenzoKoestler** [louislam/uptime-kuma#3617]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

Hey everyone, I have properly bound the socket.dock volume to Uptime Kuma (tried both /var/run/docker.sock as well as /app/data/docker.sock) and always get "Invalid URL" as response when testing in the Settings panel. 
I don't know how to proceed as it has always worked this way for any other container.
Therefore, I would classify it as a bug.

### 👟 Reproduction steps

Testing the Socker.dock connection via the settings panel

### 👀 Expected behavior

Successful access to the socket

### 😓 Actual Behavior

Error: Invalid URL

### 🐻 Uptime-Kuma Version

1.23.0

### 💻 Operating System and Arch

Linux/QNAP

### 🌐 Browser

Safari

### 🐋 Docker Version

20.10.22

### 🟩 NodeJS Version

16.20.2

### 📝 Relevant log output

```shell
TypeError [ERR_INVALID_URL]: Invalid URL
    at new NodeError (node:internal/errors:387:5)
    at URL.onParseError (node:internal/url:565:9)
    at new URL (node:internal/url:641:5)
    at Function.getHttpsAgentOptions (/app/server/docker.js:144:24)
    at Function.testDockerHost (/app/server/docker.js:84:57)
    at Socket.<anonymous> (/app/server/socket-handlers/docker-socket-handler.js:56:43)
    at Socket.emit (node:events:513:28)
    at Socket.emitUntyped (/app/node_modules/socket.io/dist/typed-events.js:69:22)
    at /app/node_modules/socket.io/dist/socket.js:703:39
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {
  input: 'undefined',
  code: 'ERR_INVALID_URL'
}
```

**Comment by CommanderStorm** [comment]
A fix for your bug was merged in https://github.com/louislam/uptime-kuma/pull/3606 and will be released in `1.23.1`

This is a duplicate of https://github.com/louislam/uptime-kuma/issues/3605

**Comment by CommanderStorm** [comment]
Please close this PR, as I cannot close it via the PR anymore

**Comment by VincenzoKoestler** [comment]
> A fix for your bug was merged in #3606 and will be released in `1.23.1`
> 
> This is a duplicate of #3605

ah! my mistake, you're amazing, thanks a ton!
